ELSpace.xcodeproj/project.pbxproj

```diff
@@ -9,10 +9,16 @@
 /* Begin PBXBuildFile section */
 		27BE1933A63FE09C3C00D679 /* Pods_ELSpaceTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 85CAC09C2A97E9B29BF92420 /* Pods_ELSpaceTests.framework */; };
 		358D6C6C62E8C93D9E41F5BF /* Pods_ELSpace.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = A7A380DA305A8B49E56FD215 /* Pods_ELSpace.framework */; };
+		500BC9771EE7EF920060D31B /* ScreenBase.swift in Sources */ = {isa = PBXBuildFile; fileRef = 500BC9761EE7EF920060D31B /* ScreenBase.swift */; };
+		500BC97C1EE7F7380060D31B /* ScreenCoordinator.swift in Sources */ = {isa = PBXBuildFile; fileRef = 500BC97B1EE7F7380060D31B /* ScreenCoordinator.swift */; };
 		5048F2441EDEE7A2001A487E /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5048F2431EDEE7A2001A487E /* AppDelegate.swift */; };
 		5048F2461EDEE7A2001A487E /* MainViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5048F2451EDEE7A2001A487E /* MainViewController.swift */; };
 		5048F24B1EDEE7A2001A487E /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 5048F24A1EDEE7A2001A487E /* Assets.xcassets */; };
 		5048F2591EDEE7A2001A487E /* ELSpaceTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5048F2581EDEE7A2001A487E /* ELSpaceTests.swift */; };
+		5063AE191EE7003A00D4E45A /* R.generated.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5063AE181EE7003A00D4E45A /* R.generated.swift */; };
```

Comment:
R.generated should not be commited to the repo. Only generated and used locally.